src/morse/index.js

```diff
@@ -58,6 +58,26 @@ const MORSE_CODE = {
 
 Object.freeze(MORSE_CODE);
 
-function morse(text) {}
+function morse(text) {
+  if (text === '') {
+    return ''
+  }
```

Comment:
Why not just do a comprehensive check here that `text` is what you want. Then check later if it's an empty string. That's easier than waiting to trim it and catching the error like you do later in the program.